### PR TITLE
docs(zone.js): fix typos and align formatting

### PR DESCRIPTION
**Pull request by fmalcher**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [x] Other... Please describe: Formatting, typos, alignment

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Review by sonukapoor — APPROVED** [review]

**Review by JiaLiPassion — APPROVED** [review]
Thank you for the document enhancement, LGTM!

**Review by aikithoughts — APPROVED** [review]
Looks good to me from a grammar/copyedit standpoint!

File: aio/content/guide/zone.md [hunk 1]
```diff
@@ -66,26 +66,26 @@ To clarify how changes are detected and values updated, consider the following c
...
       // call detectChange manually
       detectChange();
     }, 100);
 
-    // example 5: some other asynchronous APIs
+    // Example 5: some other asynchronous APIs
     document.getElementById('canvas').toBlob(blob => {
       // update value when blob data is created from the canvas
-      value = `value updated by canvas, size is ${blog.size}`;
+      value = `value updated by canvas, size is ${blob.size}`;
```
Comment (gkalpak) [inline aio/content/guide/zone.md]: Good spot!
<sub>I wish we didn't have hard-coded code examples in this guide 😞</sub>

**Review by gkalpak — APPROVED** [review]
:+1:

**Comment by gkalpak** [comment]
_Rebased on master to get CI green._

**Pull request closed without merging**